tests/unit/multi_table/test_base.py

```diff
@@ -659,3 +660,86 @@ def test_get_learned_distributions_raises_an_error(self):
         )
         with pytest.raises(ValueError, match=error_msg):
             instance.get_learned_distributions('nesreca')
+
+    def test_add_constraints(self):
+        """Test a list of constraits can be added to the synthesizer."""
+        # Setup
+        metadata = get_multi_table_metadata()
+        instance = BaseMultiTableSynthesizer(metadata)
+        positive_constraint = {
+            'constraint_class': 'Positive',
+            'table_name': 'nesreca',
+            'constraint_parameters': {
+                'column_name': 'id_nesreca',
+                'strict_boundaries': True
+            }
+        }
+        negative_constraint = {
+            'constraint_class': 'Negative',
+            'table_name': 'oseba',
+            'constraint_parameters': {
+                'column_name': 'id_nesreca',
+                'strict_boundaries': False
+            }
+        }
+
+        # Run
+        instance.add_constraints([positive_constraint, negative_constraint])
+
+        # Assert
+        positive_constraint.pop('table_name')
+        negative_constraint.pop('table_name')
+        assert instance._table_synthesizers['nesreca'].get_constraints() == [positive_constraint]
+        assert instance._table_synthesizers['oseba'].get_constraints() == [negative_constraint]
```

Comment:
Can we re-write this as a dict. The reason why is that if `positive_constraint` is being modified down the line and it's not stored as expected we may not capture it.